drafts/2019-11-06-1900011.adoc

```diff
@@ -83,6 +84,8 @@ You can now also try out early implementations of some new capabilities in the l
 
 * <<hc21, Provide your own health check procedures>>
 * <<hm21,Monitor microservice applications easily wth metrics>>
```

Comment:
Add to the end of the heading: "(MicroProfile Metrics 2.1)" (check that's the right version number re the issue link I sent you)